### ISSUE TITLE
CardFooter not rendering children without attribute "small"

### ISSUE THREAD

**Issue by NelsonRonin** [mdbootstrap/mdb-react-ui-kit#98]
In the newest version of the **CardFooter** it does not render any given child as long as you do not add the attribute `small={true}`.  
  
In the commit `b2cf40d482d424ec58a644ec04664aa1f6ddcd52` the attribute `children`  is directly used to put it into the small tag (`<small> {children} </small>`).   
Because of this change the children are not included in the `{...attributes}` and will not be rendered in the CardFooter without attribute "small".
  
Example that does **not** show footer children (here the button is not shown):   
```
<MDBCard>
    <MDBCardBody>
        Some Body Content
    </MDBCardBody>
    <MDBCardFooter>
        <MDBBtn>Some Button</MDBBtn>
    </MDBCardFooter>
</MDBCard>
```

Example that shows footer children (button will be shown): 
```
<MDBCard>
        <MDBCardBody>
            Some Body Content
        </MDBCardBody>
        <MDBCardFooter small={true}>
            <MDBBtn>Some Button</MDBBtn>
        </MDBCardFooter>
    </MDBCard>
```

**Comment by pglejzer** [comment]
Yes, this is a bug and we already fixed it. The fix version will be available on Monday. If you want you can do it for your own. You have to add props children in this component.

**Comment by pglejzer** [comment]
Yes, this is a bug and we already fixed it. The fix version will be available on Monday. If you want you can do it on your own. You have to add props children to this component.

**Comment by pglejzer** [comment]
Fixed in the [4.25.0 ](https://github.com/mdbootstrap/React-Bootstrap-with-Material-Design/releases/tag/4.25.0) version.
